.github/workflows/test.yml

```diff
@@ -27,7 +27,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        jvm-version: [17]
+        jvm-version: [ 17 ]
```

Comment:
I can't ultimately downgrade this to 11 because of https://github.com/spring-projects/spring-boot/issues/33365